docs/pages/blog/2020.md

```diff
@@ -0,0 +1,115 @@
+- Improve customizability. Add new powers to the theme with dynamic color & [variant](https://next.material-ui.com/customization/typography/#adding-amp-disabling-variants) support. Add a new [`sx` prop](https://next.material-ui.com/system/basics/) for quick customizations to all the components. Expose global class names. Deprecate the `makeStyles` and `withStyles` API.
+- Breaking changes on the API to make it more intuitive.
+- Add full support for React strict mode. We recommend to enable it.
+- Improve the performance of the system by a x3-x5 [factor](https://github.com/mui-org/material-ui/issues/21657#issuecomment-707140999).
+- Reduce bundle size: split IE11 into a different bundle [-6kB](https://github.com/mui-org/material-ui/pull/22814#issuecomment-700995216), migrate to emotion [-5kB](https://github.com/mui-org/material-ui/pull/23308#issuecomment-718748835), Popper.js v2 upgrade [-700B](https://github.com/mui-org/material-ui/pull/21761#issuecomment-657135498).
+- Improve the documentation website: search shortcut, page rating, fast material icons copy button, etc.
+
+And [much more](https://github.com/mui-org/material-ui/issues/20012).
+
+### A second theme
+
+While Material Design is a predominant design choice for building rich client-side applications, it's not in the landscape of landing and marketing pages.
+This use case for React has yet to win a large-scale adoption by the community but the traction of Next.js is a strong signal that it has the potential to happen. React pushing [server-side components](https://reactjs.org/blog/2020/12/21/data-fetching-with-react-server-components.html), with zero bundle size overhead could accelerate it.
+
+At the same time, the design trend of the latest hot startups isn't close to Material Design.
+
+We plan to provide a [second theme](https://github.com/mui-org/material-ui/issues/22485) to perfectly answer these two problems, striking a balance between:
+
+- simplicity of the design for simpler customizations
+- alignment with the best-looking design trends of hot startups
+
+It will be built on the unstyled components.
+
+### Unstyled
+
+While the completion of the unstyled components was originally part of the v5 milestone.
+We will likely finish this effort independently.
+Outside of the requirement to introduce breaking changes on the component customization API, e.g. from `PaperProps` to the `componentsProps`, [RFC #20012](https://github.com/mui-org/material-ui/issues/21453), we can work on unstyled at the same time we make progress with the second theme. The two efforts should go hand in hand.
+
+### Scale
+
+If we sustain the current growth rate, we should be able to double the size of the team, from 6 to over 10 members in the company without raising. While we will focus on strengthening all the efforts that we have started, it should start to give us room to take on significantly more ambitious problems guided by the following foundations:
```

Comment:
> from 6 to over 10

That isn't double 😄

Reply:
It's to leave more room for imagination 😇